pandas/core/ops.py

```diff
@@ -906,8 +1056,12 @@ def _make_flex_doc(op_name, typ):
 
     if typ == 'series':
         base_doc = _flex_doc_SERIES
-        doc = base_doc.format(desc=op_desc['desc'], op_name=op_name,
-                              equiv=equiv, reverse=op_desc['reverse'])
+        doc_no_examples = base_doc.format(desc=op_desc['desc'],
+            op_name=op_name, equiv=equiv, reverse=op_desc['reverse'])
```

Comment:
I think this is what failed the build. Should be aligned with the open paren on the previous line, or move `desc=...` down here.